uint8 device_id
float32 position
float32 velocity
